pages/index.js

```diff
@@ -10,23 +11,21 @@ export default function Home() {
         <img src={'/LogoDragon.svg'} width="35px" />
         RPG Sheet-Creator
       </Title>
-      <InfoContainer>
-        <h3>Welcome to RPG Sheet-Creator!</h3>
-        <p>
-          Update-News: <br />
-          You can create a Character and can write whatever you want. You can
-          edit it and also delete it. Have fun to try out! :) The Buttons in the
-          Homepage will be replaced in next time with the special new feature!
-          There will be a App Logo soon!
-        </p>
-      </InfoContainer>
       <HomeContainer>
-        <Link passHref href="/create-character">
-          <a>Create Character</a>
-        </Link>
-        <Link passHref href="/characters">
-          <a>Characters</a>
-        </Link>
+        <InfoContainer>
+          <h3>Welcome to RPG Sheet-Creator!</h3>
+          <p>
+            Create: create your Character-Sheet.
+            <br />
+            <br />
```

Comment:
Never use `<br />` tags for spacing. Make use of `<p>` tags and/or `margin`/`padding`